src/FixedDataTableRow.js

```diff
@@ -295,11 +301,21 @@ class FixedDataTableRowImpl extends React.Component {
   };
 
   _onMouseEnter = (/*object*/ event) => {
+    /**
+     * This is necessary so that onMouseLeave is fired with the initial
+     * row index since this row could be updated with a different index
+     * when scrolling.
+     */
+    this.mouseLeaveIndex = this.props.index;
```

Comment:
Don't we need to reset this onMouseOut?

Reply:
Yes, we are resetting this in _onMouseLeave right after calling this.props.onMouseLeave.

Reply:
mouseLeave isn't always fire though.
Two possible paths: 
mouseEnter -> mouseOut
mouseEnter -> mouseLeave

Reply:
In relation to the first path: why would mouseOut be fired? On the fixed data table row div we are binding specifically to the mouseLeave event.

For the second path, if mouseEnter is fired first and then mouseLeave, wouldn't the _onMouseLeave function be called  and the mouseLeaveIndex reset to null as a consequence?

Reply:
Ah sorry, your right.